app_dart/lib/src/service/build_status_service.dart

```diff
@@ -77,6 +83,18 @@ interface class BuildStatusService {
       }
     }
 
+    if (failingTasks.isEmpty && _config.flags.allowManualTreeClosures) {
```

Comment:
I'd want to see Manual Closure in front of any failing tasks if its enabled (so its less of a surprise on the dashboard). So consider moving this aboveline 73 and always adding to the failingTasks list if manual closed. The list should then look like:

['Manual Closure: blah', 'Linux tool_tests_general']

Reply:
Donezuo.